src/presentational-components/shared/breadcrubms.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+
+const RbacBreadcrumbs = (breadcrumbs) => {
+  return (
+    breadcrumbs ?  <Breadcrumb style={ { minHeight: 18 } }>
+      { Object.keys(breadcrumbs).map(item => (
```

Comment:
You can use `Object.values` since you are never actually using the object key other than to get the value. A bit more efficient approach.

Reply:
done